source/sessions/snapshot-sessions.rst

```diff
@@ -185,20 +185,20 @@ distinct response object it sends to the driver.
         atClusterTime : <BsonTimestamp>
     }
 
-The ``atClusterTime`` MUST be stored in the ``ClientSession`` to later be passed as the
-``atClusterTime`` field of the ``readConcern`` with ``snapshot`` level field  in subsequent read operations.
+The ``atClusterTime`` timestamp MUST be stored in the ``ClientSession`` to later be passed as the
+``atClusterTime`` field of a ``readConcern`` with a ``snapshot`` level in subsequent read operations.
 
 Server Errors
 =============
-1. The server may reply to read commands with a ``SnapshotTooOld`` error if the client's ``atClusterTime`` value is not available in the server's history.
-2. The server will return ``InvalidOptions`` error if both ``atClusterTime`` and ``afterClusterTime`` options are set to true.
-3. The server will return ``InvalidOptions`` error if the command does not support readConcern.level "snapshot".
+1. The server may reply to read commands with a ``SnapshotTooOld(239)`` error if the client's ``atClusterTime`` value is not available in the server's history.
+2. The server will return ``InvalidOptions(72)`` error if both ``atClusterTime`` and ``afterClusterTime`` options are set to true.
+3. The server will return ``InvalidOptions(72)`` error if the command does not support readConcern.level "snapshot".
```

Comment:
Pulled these numeric codes from https://github.com/mongodb/mongo/blob/master/src/mongo/base/error_codes.yml. I don't think the names are very useful on their own, since driver tests would ideally check for codes.

Reply:
SGTM.